src/actions/AppActions.js

```diff
@@ -462,3 +468,51 @@ export function userSessionUpdate(me) {
         me
     }
 }
```

Comment:
This piece of code was probably reintroduced by accident while rebasing with master, since you don‘t reference it in this PR. Could you please amend your commit to remove it again? I refactored it in https://github.com/metasfresh/metasfresh-webui-frontend/commit/c9e551686d00da823ddf793cd96f399a6ff006fe. Please don't copy code without leaving any reference or contribution to the authors. In this case our code broke, due to relying on undocumented React internals.

Reply:
@wiadev  can u pls fix it?